plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -85,6 +85,7 @@ public class IcebergConfig
     private Set<String> queryPartitionFilterRequiredSchemas = ImmutableSet.of();
     private int splitManagerThreads = Runtime.getRuntime().availableProcessors() * 2;
     private boolean incrementalRefreshEnabled = true;
+    private boolean metadataCacheEnabled = true;
```

Comment:
wouldn't it make also sense to do similar for delta?

Reply:
potentially yes, but out of scope, needs to be explored independently